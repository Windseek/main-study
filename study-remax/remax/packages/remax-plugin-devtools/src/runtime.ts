import { connectToDevTools } from '@remax/react-devtools-core';
import WebSocket from './WebSocket';

export default {
  onAppConfig({ config }: { config: any }) {
    const originalLaunch = config.onLaunch;
    config.onLaunch = function (...args: any) {
      try {
        console.log('尝试连接 React DevTools，请忽略连接错误信息，详情请参考 https://remaxjs.org/guide/basic/devtools');
        connectToDevTools({
          websocket: new WebSocket('ws://127.0.0.1:8097'),
        });
      } catch (e) {
        // ignore
      }
      originalLaunch.apply(config, args);
    };

    return config;
  },
};
