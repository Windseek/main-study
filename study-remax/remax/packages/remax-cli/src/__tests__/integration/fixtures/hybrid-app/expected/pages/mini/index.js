require('./../../runtime.js');
(my["webpackJsonp"] = my["webpackJsonp"] || []).push([[2],{

/***/ 6:
/***/ (function(module, exports, __webpack_require__) {

module.exports = __webpack_require__(7);


/***/ }),

/***/ 7:
/***/ (function(module, exports, __webpack_require__) {

__webpack_require__(8);

Page({});

/***/ }),

/***/ 8:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ })

},[[6,0]]]);