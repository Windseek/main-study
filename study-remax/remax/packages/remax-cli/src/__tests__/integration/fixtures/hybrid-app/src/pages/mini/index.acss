.mini {
  color: blue;
  background-image: url("/assets/bg.png");
}
