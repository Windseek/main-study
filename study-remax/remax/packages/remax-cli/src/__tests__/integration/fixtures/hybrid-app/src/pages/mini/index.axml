<import-sjs name="h" from="./index.sjs"/>
<import src="./_msg.axml"/>

<template name="msg" />
<view class="mini">{{h.upperCase(mini page)}}<view>
<image src="/assets/logo.png" />
<include src="./_footer.axml" />
