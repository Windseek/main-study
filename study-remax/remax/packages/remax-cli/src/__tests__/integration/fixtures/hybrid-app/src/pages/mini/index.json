{
  "usingComponents": {
    "foo": "/components/foo/index",
    "zxc": "../../components/zxc/index",
    "abc": "abc/index"
  }
}
