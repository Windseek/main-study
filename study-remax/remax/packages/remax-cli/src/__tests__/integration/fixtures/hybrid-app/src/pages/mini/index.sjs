const upperCase = s => s.toUpperCase();
export default {
  upperCase,
};
