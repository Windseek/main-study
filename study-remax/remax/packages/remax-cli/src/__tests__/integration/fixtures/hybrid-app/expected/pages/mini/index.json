{
  "usingComponents": {
    "foo": "/components/foo/index",
    "zxc": "/components/zxc/index",
    "abc": "/npm/abc/index"
  }
}